command/runner.go

```diff
@@ -203,3 +223,67 @@ func (r *cmdWaitReadCloser) Close() error {
 	r.closed = true
 	return nil
 }
+
+// Close the pipe and wait until the consumer program exits
+func (p pagerPipe) Close() error {
+	err := p.writer.Close()
+	<-*p.cc
+	return err
+}
+
+func (p pagerPipe) Write(b []byte) (int, error) {
+	return p.writer.Write(b)
```

Comment:
Wrapping the writer works, but I suggest returning the writer as-is instead.

There's a lot of reflection code in Go that optimizes based on what kind of writer you have.

Like, does it implement `io.Writer`, or also `io.WriterTo`, or maybe `io.WriterAt`, etc

So I suggest doing this:

```go
func (p pagerPipe) Writer() io.Writer {
	return p.writer
}
```